src/main/java/gov/nih/nci/evs/api/util/ConceptUtils.java

```diff
@@ -605,6 +613,32 @@ public static void limitQualMap(Map<String, Set<String>> qualMap, final int maxS
     }
   }
 
+  /**
+   * Intersection.
+   *
+   * @param s1 the s 1
+   * @param s2 the s 2
+   * @return the sets the
+   */
```

Comment:
Thank you.  This was using Google Guava, but I wanted to get rid of that, so apache commons is better. Unfortunately the version of this we have works with "collection" instead of "set" so it's still annoyingly needing casting or whatever.  I'll make sure to look into this in the future.  Thanks for pointing it out.  this is *great* code reveiw.